srcpkgs/Clightd/template

```diff
@@ -1,19 +1,23 @@
 # Template file for 'Clightd'
 pkgname=Clightd
-version=4.2
+version=5.1
 revision=1
 build_style=cmake
 cmake_builddir=build
 configure_args="-DENABLE_DDC=1 -DENABLE_GAMMA=1 -DENABLE_DPMS=1 -DENABLE_SCREEN=1"
 hostmakedepends="pkg-config"
-makedepends="eudev-libudev-devel libmodule-devel polkit-devel elogind-devel libXrandr-devel libXext-devel libX11-devel ddcutil"
+makedepends="eudev-libudev-devel libmodule-devel polkit-devel elogind-devel libXrandr-devel libXext-devel libX11-devel ddcutil libjpeg-turbo-devel wayland-devel libdrm-devel libusb-devel"
 depends="ddcutil"
 short_desc="API to read webcam brightness & alter screen brightness/temperature"
 maintainer="Dominic Monroe <monroef4@googlemail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/FedeDP/Clightd"
 distfiles="https://github.com/FedeDP/Clightd/archive/${version}.tar.gz"
-checksum=39301b0cad2da9adb4b662b64563f35ad5b3724631d81c1db55c311bc10dec67
+checksum=ce5002ed7790fce64e323bd483d8e5dde70a7fa9bbebc8311f514e0d5a64ab10
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" wayland-devel"
+fi
```

Comment:
You can add this unconditionally to hostmakedeps, if you want. It's likely necessary due to wayland-scanner.